VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -216,15 +215,18 @@
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:orientation="vertical"
-                    android:gravity="center_vertical|end|right">
+                    android:gravity="center_vertical|end"
```

Comment:
My point was that `end` is a superset of `right`, just as `start` is a superset of `left`.